src/types/Module.ts

```diff
@@ -78,6 +78,7 @@ export type ModuleDeliverable = {
   dueDate: DateTime;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 69]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Sorry this was a lot of boilerplate. 

```
type: string
 'type(raw)': DeliverableTypeType
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2748?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Add if you don't mind, could you also update
src/types/ProjectToDo.ts line 85?

```
this.type = searchResult['type(raw)'];
```

Should prevent the same typing error from happening there as well.

Reply:
@tommylau523 you bet, will update 👍